docs/index.fsx

```diff
@@ -1,42 +1,61 @@
 ﻿(*** condition: prepare ***)
-#I "../tests/DiffSharp.Tests/bin/Debug/net5.0"
+#I "../src/DiffSharp.Core/bin/Debug/netstandard2.1"
+#I "../src/DiffSharp.Data/bin/Debug/netstandard2.1"
+#I "../src/DiffSharp.Backends.Reference/bin/Debug/netstandard2.1"
+#I "../src/DiffSharp.Backends.Torch/bin/Debug/net5.0"
 #r "DiffSharp.Core.dll"
 #r "DiffSharp.Data.dll"
 #r "DiffSharp.Backends.Reference.dll"
 #r "DiffSharp.Backends.Torch.dll"
+// These are needed to make fsdocs --eval work. If we don't select a backend like this in the beginning, we get erratic behavior.
+DiffSharp.dsharp.config(backend=DiffSharp.Backend.Reference)
+DiffSharp.dsharp.seed(123)
+
 (*** condition: fsx ***)
 #if FSX
 #r "nuget: DiffSharp-lite,{{fsdocs-package-version}}"
 #endif // FSX
 (*** condition: ipynb ***)
 #if IPYNB
+// Google Colab only: uncomment and run the following to install dotnet and the F# kernel
+// !bash <(curl -Ls https://raw.githubusercontent.com/gbaydin/scripts/main/colab_dotnet5.sh)
+#endif // IPYNB
+(*** condition: ipynb ***)
+#if IPYNB
+// Import DiffSharp package
 #r "nuget: DiffSharp-lite,{{fsdocs-package-version}}"
 
+// Set dotnet interactive formatter to plaintext
 Formatter.SetPreferredMimeTypeFor(typeof<obj>, "text/plain")
 Formatter.Register(fun (x:obj) (writer: TextWriter) -> fprintfn writer "%120A" x )
 #endif // IPYNB
 
 (**
+[![Binder](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/DiffSharp/diffsharp.github.io/blob/master/{{fsdocs-source-basename}}.ipynb)&emsp;
 [![Binder](img/badge-binder.svg)](https://mybinder.org/v2/gh/diffsharp/diffsharp.github.io/master?filepath={{fsdocs-source-basename}}.ipynb)&emsp;
 [![Script](img/badge-script.svg)]({{fsdocs-source-basename}}.fsx)&emsp;
 [![Script](img/badge-notebook.svg)]({{fsdocs-source-basename}}.ipynb)
 
 # DiffSharp: Differentiable Tensor Programming Made Simple
 
-DiffSharp is a tensor library with support for [differentiable programming](https://en.wikipedia.org/wiki/Automatic_differentiation).
+DiffSharp is a tensor library with support for [differentiable programming](https://en.wikipedia.org/wiki/Differentiable_programming).
 It is designed for use in machine learning, probabilistic programming, optimization and other domains.
 
+<button class="button" style="vertical-align:middle" onclick="window.location.href='{{root}}install.html'"><span>Install »</span></button>
+
+## Key Features
+
 🗹 Nested and mixed-mode differentiation
 
 🗹 Common optimizers, model elements, differentiable probability distributions
 
-🗹 F# for robust functional programming 
+🗹 F# programming language for robust functional programming 
```

Comment:
The repeated 'programming' doesn't look right, I think the former is better?

Reply:
I agree, we can revert. I was thinking about people read "F#" for the first time in their lives.